js/src/values/defaultReplaceVariables.js

```diff
@@ -2,90 +2,112 @@
 const defaultReplaceVariables = [
 	{
 		name: "currentdate",
+		label: "Current date",
 		value: "",
 	},
 	{
 		name: "currentday",
+		label: "Current day",
 		value: "",
 	},
 	{
 		name: "currentmonth",
+		label: "Current month",
 		value: "",
 	},
 	{
 		name: "currenttime",
+		label: "Current time",
 		value: "",
 	},
 	{
 		name: "currentyear",
+		label: "Current year",
 		value: "",
 	},
 	{
 		name: "date",
+		label: "Date",
 		value: "",
 	},
 	{
 		name: "id",
+		label: "Post ID",
 		value: "",
 	},
 	{
 		name: "page",
+		label: "Page number",
 		value: "",
 	},
 	{
 		name: "searchphrase",
+		label: "Search phrase",
 		value: "",
 	},
 	{
 		name: "sitedesc",
+		label: "Tagline",
 		value: "",
 	},
 	{
 		name: "sitename",
+		label: "Site name",
 		value: "",
 	},
 	{
 		name: "category",
+		label: "Category",
 		value: "",
 	},
 	{
 		name: "focuskw",
+		label: "Keyword",
 		value: "",
 	},
 	{
 		name: "title",
+		label: "Page name",
 		value: "",
 	},
 	{
 		name: "parent_title",
+		label: "Parent title",
 		value: "",
 	},
 	{
 		name: "excerpt",
+		label: "Excerpt",
 		value: "",
 	},
 	{
 		name: "primary_category",
+		label: "Primary category",
 		value: "",
 	},
 	{
 		name: "sep",
+		label: "Separator",
 		value: "",
 	},
 	{
 		name: "excerpt_only",
+		label: "Excerpt only",
 		value: "",
 	},
 	{
 		name: "category_description",
+		label: "Category description",
 		value: "",
 	},
 	{
 		name: "tag_description",
+		label: "Tag description",
 		value: "",
 	},
 	{
 		name: "term_description",
+		label: "Term description",
 		value: "",
 	},
 ];
```

Comment:
For better readability, I'd recommend to re-order all these variables sorting them alphabetically.